docs/api/wait_strategies.md

```diff
@@ -10,3 +10,22 @@ _ = Wait.ForUnixContainer()
   .UntilOperationIsSucceeded(() => true, 1)
   .AddCustomWaitStrategy(new MyCustomWaitStrategy());
 ```
+
+## Wait for healthy container
```

Comment:
Due to nested code block I cannot add a suggestion (at least I do not know how it works 😅). WDYT about something like:

````
## Wait until the container is healthy

If the Docker image supports Dockers's [HEALTHCHECK](docker-docs-healthcheck) feature, like the following configuration:

```Dockerfile
FROM alpine
HEALTHCHECK --interval=1s CMD test -e /healthcheck
```

You can leverage the container's health status as your wait strategy to report readiness of your application or service:

```csharp
_ = new TestcontainersBuilder<TestcontainersContainer>()
  .WithWaitStrategy(Wait.ForUnixContainer().UntilContainerIsHealthy())
  .Build();
```

[docker-docs-healthcheck]: https://docs.docker.com/engine/reference/builder/#healthcheck
````